src/validation.cpp

```diff
@@ -1789,10 +1791,11 @@ bool CChainState::ConnectBlock(const CBlock &block, BlockValidationState &state,
         int nSigChecksRet;
         TxValidationState tx_state;
         if (fScriptChecks &&
-            !CheckInputScripts(tx, tx_state, view, flags, fCacheResults,
-                               fCacheResults, PrecomputedTransactionData(tx),
-                               nSigChecksRet, nSigChecksTxLimiters[txIndex],
-                               &nSigChecksBlockLimiter, &vChecks)) {
+            !CheckInputScripts(
```

Comment:
Is the view here going to be able to handle transactions that are chained in the same block?

Reply:
I'm pretty sure CCoinsView handles it. Not sure how to test it at this moment

Reply:
Answered my question. Up on line 1704 the outputs are all added to the view are all added to be able to check the inputs for CTOR.

Reply:
Shammah [Not asking for crypto], [3/18/21, 8:42:24 PM]:
> line 1704 answers my question
> The outputs are all added to the view before checking anything